websites/Z/Zalo/metadata.json

```
@@ -0,0 +1,45 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.15",
+  "apiVersion": 1,
+  "author": {
+    "id": "379076400394666004",
+    "name": "bcraftmg"
+  },
+  "service": "Zalo",
+  "description": {
+    "en": "Zalo is a Vietnamese instant messaging multi-platform service developed by VNG Corporation",
+    "vi": "Zalo là một ứng dụng nhắn tin nhanh đa nền tảng của Việt Nam, được phát triển bởi VNG"
+  },
+  "url": "chat.zalo.me",
+  "version": "1.0.0",
```

We're requiring a regexp in preparation for future updates. The `url` property will be purely decorative in the future.
```suggestion
  },
  "url": "chat.zalo.me",
  "regExp": "//chat[.]zalo[.]me/
  "version": "1.0.0",
```